cmd/lifecycle/platform/v06/platform.go

```diff
@@ -0,0 +1,27 @@
+package v06
+
+import (
+	"github.com/buildpacks/lifecycle"
+	"github.com/buildpacks/lifecycle/api"
+	"github.com/buildpacks/lifecycle/cmd/lifecycle/platform/common"
+)
+
+type v06Platform struct {
+	api              *api.Version
+	previousPlatform common.Platform
+}
+
+func NewPlatform(previousPlatform common.Platform) common.Platform {
+	return &v06Platform{
+		api:              api.MustParse("0.6"),
+		previousPlatform: previousPlatform,
+	}
+}
+
+func (p *v06Platform) API() string {
+	return p.api.String()
+}
+
+func (p *v06Platform) AnalyzeOperations() []lifecycle.AnalyzeOperation {
+	return p.previousPlatform.AnalyzeOperations()
```

Comment:
I like the way this reads
```suggestion
	return []lifecycle.AnalyzeOperation{v05.AnalyzeOperations()...}
```
I also like the idea that we could interject a new step if we make the operations public.

```suggestion
	return []lifecycle.AnalyzeOperation{v05.ReadPreviousImage, DoSomethingNewForThisVersion(), v05.RestoreLayerMetadata()}
```

Reply:
Changed in 7899b43